vllm/worker/hpu_model_runner.py

```diff
@@ -77,14 +78,42 @@
 # Use caution when updating them!
 _PAD_SLOT_ID = 0
 _PAD_BLOCK_ID = 0
-
+_UNSET_NUM_PATCHES = 9999999
 LORA_WARMUP_RANK = 8
 
 VLLM_DELAYED_SAMPLING = os.environ.get('VLLM_DELAYED_SAMPLING',
                                        'false').lower() == 'true'
 VLLM_MERGED_PREFILL = os.environ.get('VLLM_MERGED_PREFILL',
                                      'false').lower() == 'true'
 DUMMY_TOKEN_ID = -1
+'''
+This class is used to bucket image tokens
+'''
+
+
+class VisionBuckets():
+
+    def __init__(self):
+        envvar = os.environ.get('VLLM_MULTIMODAL_BUCKETS', "")
+        if envvar == "":
+            #TODO:with profile_run, the bucket of 65536 is added, so the pixel values
```

Comment:
this statement is no longer true i think. we profile with largest bucket in this class